src/types/styles.ts

```diff
@@ -154,4 +176,6 @@ export interface StyleSettings {
   useCoolingModule: boolean;
   coolingModuleSpeed: number;
   enableCoolingModuleAtLayer: number;
+  // Defaults will be applied in opal-tide if undefined.
```

Comment:
I think this comment isn't necessary, as the same is true for pretty much all other settings